src/less/parse.js

```diff
@@ -182,11 +182,15 @@ function getLastPositionForArray(content, line, column, colOffset) {
 
   if (!colOffset) return position;
 
-  if (tokens[pos - 1].type !== 'Newline') {
-    position[1] += colOffset;
+  if (tokens[pos - 1]) {
+    if (tokens[pos - 1].type !== 'Newline') {
+      position[1] += colOffset;
+    } else {
+      position[0]++;
+      position[1] = 1;
+    }
   } else {
-    position[0]++;
-    position[1] = 1;
+    position[1] = colOffset;
```

Comment:
tokens[pos - 1] is undefined while tests run. under that circumstance, the column position is equal to the colOffset for a variable dictionary. tested that with real-world examples (as the test less files aren't really) and it checked out.